### PR TITLE
update to latest supabase-js lib version

### PR DESCRIPTION
**Pull request by romanetar**
ref https://tipit.avaza.com/project/view#!tab=task-pane&groupby=MyTaskDueDate&view=vertical&task=3042730&fileview=grid

### CODE REVIEW

File: src/lib/repository/AccessRepository.js [hunk 1]
```diff
@@ -25,28 +25,30 @@ export default class AccessRepository extends AttendeeRepository {
...
 
   refreshRealtimeSubscription() {
     //const subscriptions = this._client.getSubscriptions()
-    if (this._subscription && this._subscription.state === 'joined') {
+    if (this._subscription?.state === 'joined' || this._subscription?.state === 'joining') {
       return
     }
 
-    console.log('re-subscribing to realtime...')
+    console.log('re-subscribing to realtime...', this._subscription?.state)
```
Comment (smarcet) [inline src/lib/repository/AccessRepository.js:40]: could we add context to the logs ?
like A2A::ClassName::MethodName ?

Comment (romanetar) [reply src/lib/repository/AccessRepository.js]: done

File: src/lib/repository/AccessRepository.js [hunk 2]
```diff
@@ -25,28 +25,30 @@ export default class AccessRepository extends AttendeeRepository {
...
+    console.log('_handleRTSubscriptionNews')
     if (this._newsListener) {
       this._newsListener(news)
     }
   }
 
   refreshRealtimeSubscription() {
     //const subscriptions = this._client.getSubscriptions()
-    if (this._subscription && this._subscription.state === 'joined') {
+    if (this._subscription?.state === 'joined' || this._subscription?.state === 'joining') {
```
[the remark this answers is not in the document]
Comment (romanetar) [reply src/lib/repository/AccessRepository.js]: done

**Review by smarcet — APPROVED** [review]
LGTM

**Pull request merged**